scss/base/base.scss

```diff
@@ -0,0 +1,32 @@
+* { font-family: 'Quattrocento Sans', sans-serif; }
+
+nav {margin-bottom: 3em;}
+
+h1
+{
+	color: #0e48a1;
+	font-weight: 900;
+}
+
+h2
+{
+	margin-top: 1em;
+	font-weight: 700;
+}
+
+h3, p.lead { font-weight: 600; }
+
+h3 { margin-top: 1em; }
+
+
+.lead { margin-bottom: 3em; }
+
+
+p a, li a, h3 a { color: #0e48a1; }
+
+.btn-primary { background-color: #0e48a1; }
+
+
+footer { margin: 3em 0em 3em 0em; }
+
+.embed-responsive {border: 1px solid #ced4da; border-radius: .25rem;}
```

Comment:
This is the same as `graphwidget` do you want to keep it here?

Reply:
oops fixed thank you